game/tests.py

```
@@ -1631,3 +1631,62 @@ def fake_engine(cmd):
+
+    def test_delete_account_post_invalid_credentials(self):
+        self.client.login(username='deleteme', password='Password123!')
+        response = self.client.post(reverse('delete_account'), {
+            'username': 'deleteme',
+            'password': 'WrongPassword'
+        }, follow=True)
+        self.assertContains(response, 'Invalid username or password.')
+
+    @override_settings(DEBUG=True, EMAIL_HOST_USER='', EMAIL_HOST_PASSWORD='')
+    def test_delete_account_post_success_debug(self):
+        self.client.login(username='deleteme', password='Password123!')
```

Ruff (0.15.14)

[error] Possible hardcoded password assigned to argument: "password"

(S106)